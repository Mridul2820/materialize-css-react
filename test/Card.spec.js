import React from 'react';
import { render } from '@testing-library/react';
import Card from '../src/Card';

describe('<Card />', () => {
  test('renders', () => {
    const { container } = render(
      <Card
        title="card title"
        className="blue-grey darken-1"
        textClassName="white-text"
      >
        I am a very simple card
      </Card>
    );

    expect(container).toMatchSnapshot();
  });
});
