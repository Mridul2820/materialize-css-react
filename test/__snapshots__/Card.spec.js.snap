// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders 1`] = `
<div>
  <div
    class="blue-grey darken-1 card"
  >
    <div
      class="card-content white-text"
    >
      <span
        class="card-title"
      >
        card title
      </span>
      <div>
        I am a very simple card
      </div>
    </div>
    <div
      class="card-reveal"
    >
      <span
        class="card-title"
      >
        card title
        <i
          class="material-icons right"
        >
          close
        </i>
      </span>
    </div>
  </div>
</div>
`;
